code/src/components/Date.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import moment from "moment";
+
+import styled from "styled-components";
+
+export const Date = () => {
+  const now = moment()
+    .format("LL")
+    .toString();
+  return <Container>Today: {now}</Container>;

```

Comment:
Nice work with the date formatting and also that you made this into its own component